packages/@sanity/server/README.md

```diff
@@ -0,0 +1,5 @@
+# ⚠️ THIS PACKAGE IS DEPRECATED
```

Comment:
We still have the server package in version 3 – is it correct to have a deprecation warning for this package?

Reply:
cc @rexxars

Reply:
It is going to be removed in the vite 4 PR, so I think it's fine to leave this deprecation warning until then, especially since it is not meant to be used outside of our internals.